config/locales/en.yml

```diff
@@ -1461,11 +1461,11 @@ en:
         label: Money in bank accounts
         hints:
           - For example, money in current or savings accounts, ISAs, or child accounts your client is a signatory on.
-          - Do not include any government Cost of Living Payments owed to your client.
+          - Do not include any disregarded capital or cost of living payments. Check the guidance for a complete list of disregarded capital.
```

Comment:
can we check if Cost of Living should be capitalised. This PR doesnt capitalise but EL-1520 does and it would be good to be consistent across the service.

Reply:
Sure thing @willc-work, I'll ask as a comment on the JIRA with the UAT link.

Reply:
@willc-work yes, it should be Capitalised and have now done so.